.github/workflows/test.yml

```diff
@@ -0,0 +1,69 @@
+name: Test
+
+on:
+  - push
+  - pull_request
+
+env:
+  CI: "true"
+  FORCE_COLOR: "true"
+  DEBUG: "*"
+
+jobs:
+  test:
+    name: Test Node v${{ matrix.node }} on ${{ matrix.os }} and ${{ matrix.puppeteer_version }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        puppeteer_version:
+          # - 5.0.0
+          - 2.1.1 # Chromium 79.0.3942.0, Oct 24 2019
+          # - 2.0.0 # Chromium 79.0.3942.0, Oct 24 2019
+          # - 1.20.0 # Chromium 78.0.3882.0, Sep 13 2019
+          # - 1.15.0 # Chromium 75.0.3765.0, Apr 26 2019
+          # - 1.9.0 # Chromium 71.0.3563.0, Oct 4, 2018
+          # - 1.6.2 # Chromium 69.0.3494.0, Aug 1, 2018
+        node:
+          - 10
```

Comment:
I'd suggest we test using the latest NodeJS LTS & Current release from now on (v12 & v14, maybe v10 as well if GH actions is fast):
https://nodejs.org/en/about/releases/

Reply:
Added Node 10, 12, 14: https://github.com/berstend/puppeteer-extra/actions/runs/164138527